django-backend/fecfiler/feedback/views.py

```diff
@@ -0,0 +1,43 @@
+import github3
+import structlog
+from fecfiler.settings import FECFILE_GITHUB_TOKEN
+from rest_framework import status, viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from .serializers import FeedbackSerializer
+
+logger = structlog.get_logger(__name__)
+
+
+class FeedbackViewSet(viewsets.ViewSet):
+    @action(
+        detail=False,
+        methods=["post"],
+        url_path="submit",
+    )
+    def submit_feedback(self, request):
+        serializer = FeedbackSerializer(data=request.data, context={"request": request})
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        title = "User feedback on " + serializer.validated_data["location"]
+
+        body = (
+            "## What were you trying to do and how can we improve it?\n %s \n\n"
+            "## General feedback?\n %s \n\n"
+            "## Tell us about yourself\n %s \n\n"
+            "## Details\n"
+            "* URL: %s \n"
+            "* User Agent: %s"
+        ) % (
+            serializer.validated_data["action"],
```

Comment:
Does github sanitize content to it's api?  wondering if we're responsible for removing dangerous content like scripts

Reply:
great question - I couldn't find a definitive answer for this so I added escaping on our end to the title and body.